src/components/loaders/loader.component.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { InlineLoading } from '@carbon/react';
+import styles from './loader.scss';
+
+const isLoadingPreviousEncounter: React.FC = () => {
```

Comment:
This name is kind of confusing, someone may think this is some React state that flags whether we are loading the `previousEncounter`; yet it's the loader itself. I would rename it to something like `Loader`.

Reply:
Nice catch 😅